app/src/main/java/net/programmierecke/radiodroid2/RadioDroidApp.java

```diff
@@ -91,6 +93,8 @@ public void onCreate() {
         favouriteManager = new FavouriteManager(this);
         recordingsManager = new RecordingsManager();
         alarmManager = new RadioAlarmManager(this);
+        tvChannelManager = new TvChannelManager(this);
```

Comment:
Check if running on TV hardware first (https://developer.android.com/training/tv/start/hardware.html). Also seems to crash otherwise.

Reply:
Right, thanks, I completely forgot about this. Right now it only checks the API level, which is not enough.

Reply:
Addressed in 00bd7af.